extensions/api/observability/src/main/java/org/eclipse/dataspaceconnector/api/observability/ObservabilityApiController.java

```diff
@@ -29,6 +30,7 @@
 @Consumes({ MediaType.APPLICATION_JSON })
 @Produces({ MediaType.APPLICATION_JSON })
 @Path("/check")
+@Tag(name = "Application Observability")
```

Comment:
We agreed on putting the swagger annotations on the `*Api` interface, not on the `*ApiController` implementation